### ISSUE TITLE
Unexpected token error with dynamic import()

### ISSUE THREAD

**Issue by thallada** [babel/babel-eslint#496]
I'm unable to get eslint to parse ES2015 dynamic `import()`s. I've searched for solutions, and everyone points to this library as a solution, but it doesn't work for me.

I created a minimal example at https://github.com/thallada/babel-eslint-dynamic-import which has more details.

**Comment by hzoo** [comment]
Are you on the latest version? 7.2.3 is the current `latest`

**Comment by thallada** [comment]
Ah, that was the problem. The latest version works.

I followed the README which told me to run:
```
npm install eslint@3.x babel-eslint@6 --save-dev
```
Shouldn't that be this now?
```
npm install eslint@3.x babel-eslint@7 --save-dev
```

**Comment by hzoo** [comment]
yeah I guess it's not updated - can you make a PR?
